packages/perseus-editor/src/components/start-coord-settings.tsx

```diff
@@ -20,44 +25,84 @@ import type {
 
-    // Check if coords is of type CollinearTuple
     switch (type) {
+        // coords with type CollinearTuple
         case "linear":
         case "ray":
+            const linearCoords =
+                startCoords ?? getLineCoords(props, range, step);
             return (
                 <>
                     <View style={styles.tile}>
                         <LabelLarge>Point 1</LabelLarge>
                         <CoordinatePairInput
-                            coord={startCoords[0]}
+                            coord={linearCoords[0]}
                             onChange={(value) =>
-                                onChange([value, startCoords[1]])
+                                onChange([value, linearCoords[1]])
                             }
                         />
                     </View>
                     <View style={styles.tile}>
                         <LabelLarge>Point 2</LabelLarge>
                         <CoordinatePairInput
-                            coord={startCoords[1]}
+                            coord={linearCoords[1]}
                             onChange={(value) =>
-                                onChange([startCoords[0], value])
+                                onChange([linearCoords[0], value])
                             }
                         />
                     </View>
                 </>
             );
+        // coords with type CollinearTuple[]
+        case "linear-system":
+        case "segment":
+            const multiLineCoords: CollinearTuple[] =
+                startCoords ??
+                (type === "segment"
+                    ? getSegmentCoords(props, range, step)
+                    : getLinearSystemCoords(props, range, step));
+            return (
+                <>
+                    {multiLineCoords.map((coordPair, i) => (
+                        <PerseusEditorAccordion
+                            key={`segment-${i}-start-coords`}
+                            header={
+                                <LabelLarge>{`Segment ${i + 1}`}</LabelLarge>
+                            }
+                            expanded={true}
+                        >
+                            <View style={styles.nestedTile}>
+                                <LabelLarge>Point 1</LabelLarge>
+                                <CoordinatePairInput
+                                    coord={coordPair[0]}
+                                    onChange={(value) => {
+                                        const newCoords = [...multiLineCoords];
+                                        newCoords[i] = [value, coordPair[1]];
+                                        onChange(newCoords);
+                                    }}
+                                />
+                            </View>
+                            <View style={styles.nestedTile}>
+                                <LabelLarge>Point 2</LabelLarge>
```

Comment:
This is more of a general question/curiosity than a nit with this PR: How are we thinking about labelling / handling these points when we get to things like the Polygon or Angle graphs? (Do we start in the upper left-hand corner and go clockwise?)

Reply:
I think I was just going to do it in the order that the points are saved/set in the initializer function `getPolygonCoords`. I'm not sure about labeling it, but I'll definitely experiment with it when I get to polygons.